README.md

```diff
@@ -177,6 +177,7 @@ $ curl -H "Authorization: Basic <base64_encoded_token>" http://localhost:8083/v1
 | :---: | :---: | :---: | :---: | :--- |
 | **detector-cycle-time** | string | no | `3s` | Time (in seconds) to wait between each detector cycle. |
 | **port** | string | no | `:8083` | Address to listen on for detector HTTP server.<br/> **NOTE:** If your `detector` is listening on a non-default port, don't forget to start your `aggregator` with `--detector-port` flag. This will inform `aggregator` which `detector` port to reach out to. |
+| **prometheus-metrics-path** | string | no | `/v1/metrics/` | Set the path that is used by the metrics endpoint to expose metrics in the prometheus format. |
```

Comment:
Small nitpick: Added `detector` to the description.

`Set the path that is used by the metrics endpoint to expose detector metrics in the prometheus format.`